src/Mapbender/WmsBundle/Component/WmsMetadata.php

```diff
@@ -80,8 +80,8 @@ private function prepareLayers($layer)
     {
         $layer_items = array();
         $layer_items[] = array("name" => strval($layer->getSourceItem()->getName()));
-        $layer_items[] = array("title" => strval(
-            $layer->getSourceItem()->getTitle()) . ' (' . $layer->getTitle() . ')');
+        $sourceItem = $layer->getSourceItem();
```

Comment:
Same logic as the source: show `source title (ìnstance title)` => use same method for formatting